### DISCUSSION TITLE
is it possible to support hiding sensitive data in report for pytest?

### DISCUSSION THREAD

**Discussion by nikeasyanzi** [allure-framework/allure2#1610] [Bug Reports]
I find the feature has implemented as `secret` in java.
But is there any plan to implement that in other testing tools?

Feature request: a way to disable step parameter auto-displaying in the report.
https://github.com/allure-framework/allure2/issues/863

**Comment by delatrie** [comment]
I've created a feature request in allure-python. The link: allure-framework/allure-python#805.
